composer.json

```diff
@@ -67,10 +67,10 @@
 	},
 	"scripts": {
 		"test": [
-			"phpunit -c phpunit.xml.dist"
+			"@php ./vendor/phpunit/phpunit/phpunit"
```

Comment:
Why was `-c` removed here, but not for `"integration-test"`?

Reply:
Does that answer your question ?

Reply:
@jrfnl Yes, perfectly! Thanks! 😊